StortSpelProjekt/Game/src/Gamefiles/Components/UpgradeComponents/Upgrades/UpgradeRangeBounce.cpp

```diff
@@ -0,0 +1,35 @@
+#include "UpgradeRangeBounce.h"
+#include "EngineMath.h"
+#include "ECS/Entity.h"
+#include "Components/RangeComponent.h"
+
+UpgradeRangeBounce::UpgradeRangeBounce(Entity* parentEntity) : Upgrade(parentEntity)
+{
+	SetName("UpgradeRangeBounce");
+	SetType(F_UpgradeType::PLAYER);
+
+	m_Price = 1000;
+	m_StartingPrice = m_Price;
+	m_MaxLevel = 1;
```

Comment:
A bit expensive? :( Also same as above.. is this upgarde not supposed to scale with levels? Maybe I missed it, but I cannot find anything about level-scaling in this upgrade?

Reply:
If it's only buy-able once I think it should be expensive 😁